doc/user-guide/template.md

```diff
@@ -294,3 +293,25 @@ sex = "FEMALE"
 %%sql
 SELECT * FROM one_sex
 ```
+
+## Disabling named parameters
+
+The named parameters option can be _Disabled_ using `%config SqlMagic.named_parameters = 2`
+
+Disabling named parameters allows you to execute queries with syntax which SqlAlchemy would normally mistake for bind parameters.
+
+For example, a query like : `%sql select json('[{"a":1}]')`
+
+Would return this error: `RuntimeError: (sqlalchemy.exc.InvalidRequestError) A value is required for bind parameter '1'`
+
+However, if we disable named parameters, we can execute the query successfully.
+
+```{code-cell} ipython3
+%load_ext sql
+%sql duckdb://
+%config SqlMagic.named_parameters = 2
+
+%sql select json('[{"a":1}]')
+```
+
+This option is helpful when executing queries using JSON or special DB syntax, e.g. `%sql select parse_json('{"fruit":"banana"}'):fruit` in the case of Snowflake.
```

Comment:
It would be better to add it in the Snowflake guide

Reply:
[Moved this to a new section in the Snowflake guide](https://github.com/ploomber/jupysql/pull/980/files#diff-fa53cd2419e9f982b8a8b1040c127b5c889becc9318dc2cc9904dc5e99eb06bfR762-R776)